core/pom.xml

```diff
@@ -355,6 +355,24 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.chemistry.opencmis</groupId>
+      <artifactId>chemistry-opencmis-client-api</artifactId>
+      <version>1.1.0</version>
```

Comment:
manage id root pom dependency management?

Reply:
@jodygarnett 
I'm assuming that you mean copy the pom dependency to the root pom.xml and then remove the version information from the core/pom.xml
As this implementation is similar to the S3 implementation I copied the same method that was used by the following.

```
 <dependency>
      <groupId>com.amazonaws</groupId>
      <artifactId>aws-java-sdk-s3</artifactId>
      <version>1.11.618</version>
    </dependency>
```

I also noticed that there are lots of other dependencies with versions in the core/pom.xml

If this is a new requirement for the GN project then I suggest that we do this cleanup as a new PR.
Or maybe I'm misunderstanding what you are suggesting?

Reply:
I understand best practice, is not always current practice.